### DISCUSSION TITLE
Ground placement instructions

### DISCUSSION THREAD

**Discussion by MoriTAH3110** [google/model-viewer#1825] [Q&A]
Hi everyone!

First of all, thanks to all collaborators in model-viewer for this amazing tool.

I'm currently trying to develop a web ar model viewer for my company taking advantage of the annotation feature, however, I have some issues with placing models on the ground. At first, the models appears floating in space and it like the phone you are holding on your hand is the model, so it is not placed on a surface in front on you until it detects a proper plane for placement.

I would like to know if there is a way to enable some GUI instructions, similar to the ones in Scene Viewer that tells the user to move the phone around in order to find the ground and then place the desired model.

Thanks in advance for all your help!

**Comment by elalish** [comment]
Absolutely! We even have an example of it: https://modelviewer.dev/examples/augmentedreality/#webXR

I believe what you're looking for is the `ar-status` attribute; you can see how we've used it to power the custom `#ar-prompt` element.

**Reply by MoriTAH3110** [reply]
Thank you very much for your response. I will look into this information
